.github/workflows/third_party_notices_check.yml

```diff
@@ -21,6 +23,8 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
       - run: npm install -g generate-license-file
       - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.REPO_SCOPED_TOKEN }}
```

Comment:
the token have to have repo scoped permission for the third party github action to rerun workflows. However, it also needs to be yext authorized since the repos we have are under yext org - can't push to repos otherwise. Do you perfer if a rename it to ORG_REPO_SCOPED_TOKEN?

Reply:
I wasn't sure what "repo scoped" meant. I'm not an expert in github stuff, but under the dropdown in settings I see three options, Actions, Codespaces, and Dependabot. Under Actions, there are 3 categories of secrets, Environment Secrets, Repository Secrets, and Organization Secrets.
From the naming, my hunch was that here, "repo scoped" meant that the secret had to be specifically a "Repository Secret", and env or org secrets wouldn't work. But I wasn't sure if that was true?

Reply:
ooh I see. This token is under Repository Secrets. the scope I meant is when you generate the token ([docs](https://docs.github.com/en/developers/a
<img width="475" alt="Screen Shot 2022-05-05 at 11 30 48 AM" src="https://user-images.githubusercontent.com/36055303/166958953-de035500-f9cf-4abc-bbad-0d4c073ac630.png">
pps/building-oauth-apps/scopes-for-oauth-apps)):

Reply:
and by yext authorized, I mean after the token generation, you can do this!
<img width="412" alt="Screen Shot 2022-05-05 at 11 34 12 AM" src="https://user-images.githubusercontent.com/36055303/166959469-c74792bf-b63f-4cda-be96-74a119833184.png">

Reply:
ooo